### DISCUSSION TITLE
[Bug] Messages are displayed incorrectly on iOS

### DISCUSSION THREAD

**Discussion by philippkuehn** [community/community#8713] [Mobile]
Since a few months I have the problem that messages are displayed incorrectly. This happens especially when I open new notifications for the first time. When messages are displayed incorrectly, only a restart of the app helps. I’m using iOS 15.1.1 on an iPhone 12 mini.

### Sometimes messages are overlapped

<img width="50%" src="https://user-images.githubusercontent.com/2500670/145686607-acbd4b3f-c449-40fc-bb92-19d58dae39e2.PNG" />

### Sometimes messages are cropped

<img width="50%" src="https://user-images.githubusercontent.com/2500670/145686496-3d0e18a0-b17c-4df2-8c79-7b87e91e96a3.PNG" />

### Sometimes there is weird whitespace

<img width="50%" src="https://user-images.githubusercontent.com/2500670/145686526-3a9eca9d-ef2b-406e-9336-416542badcd6.PNG" />

Here is a video

https://user-images.githubusercontent.com/2500670/145686646-2dd77079-27c4-4438-a7b9-57c0977b2ce4.MP4

**Comment by Djon79ua** [comment]
> Since a few months I have the problem that messages are displayed incorrectly. This happens especially when I open new notifications for the first time. When messages are displayed incorrectly, only a restart of the app helps. I’m using iOS 15.1.1 on an iPhone 12 mini.
> 
> 
> 
> ## Examples
> 
> 
> 
> ### Sometimes messages are overlapped
> 
> 
> 
> <img width="50%" src="https://user-images.githubusercontent.com/2500670/145686607-acbd4b3f-c449-40fc-bb92-19d58dae39e2.PNG" />
> 
> 
> 
> ### Sometimes messages are cropped
> 
> 
> 
> <img width="50%" src="https://user-images.githubusercontent.com/2500670/145686496-3d0e18a0-b17c-4df2-8c79-7b87e91e96a3.PNG" />
> 
> 
> 
> ### Sometimes there is weird whitespace
> 
> 
> 
> <img width="50%" src="https://user-images.githubusercontent.com/2500670/145686526-3a9eca9d-ef2b-406e-9336-416542badcd6.PNG" />
> 
> 
> 
> Here is a video
> 
> 
> 
> https://user-images.githubusercontent.com/2500670/145686646-2dd77079-27c4-4438-a7b9-57c0977b2ce4.MP4
